### PR TITLE
Feature: Uplift to TS26.512 v17.3.0

### PR DESCRIPTION
**Pull request by davidjwbbc**
Add overrides for TS 26.512 v17.3.0 OpenAPI YAML files before their official release.

This includes a fix for relative URL formats which is drafted for v17.4.0.

Also fixes issue with choosing the correct flag ('-noenc' or '-nodes') in the `make_self_signed_certificates.py` script.

Examples are now no longer shared and have been moved into their respective project repositories.

### CODE REVIEW

**Review by devbbc — APPROVED** [review]

File: 5gms/5G_APIs-overrides/TS26512_CommonData.yaml [hunk 1]
```diff
@@ -0,0 +1,244 @@
+openapi: 3.0.0
+info:
+  title: 5GMS Common Data Types
+  version: 2.0.1
```
Comment (rjb1000) [inline 5gms/5G_APIs-overrides/TS26512_CommonData.yaml:4]: Should probably make this "2.Y.Z" to indicate that this is a pre-standardisation proposal.

File: 5gms/5G_APIs-overrides/TS26512_CommonData.yaml [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+ 
+    RelativeUrl:
+      type: string
+      format: uri-reference
+      description: Relative Uniform Resource Locator, conforming with the "relative-ref" production specified in IETF RFC 3986, section 4.2.
+
+    AbsoluteUrl:
+      type: string
+      format: uri
+      description: Relative Uniform Resource Locator, conforming with the "absolute-url" production specified in IETF RFC 3986, section 4.3 in which the scheme part is "http" or "https".
```
Comment (rjb1000) [inline 5gms/5G_APIs-overrides/TS26512_CommonData.yaml:44]: Oops... Copy-and-paste error. This should say "Absolute Uniform Resource Locator..." of course.

File: 5gms/5G_APIs-overrides/TS26512_CommonData.yaml [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+      description: String chosen by the 5GMS AF to serve as an identifier in a resource URI.
+    Percentage:
+      type: number
+      minimum: 0.0
+      maximum: 100.0
+    #DurationSec is defined in TS29571_CommonData
+    #DateTime is defined in TS29571_CommonData
+    #Uri is defined in TS29571_CommonData
+    Url:
+      type: string
```
Comment (rjb1000) [inline 5gms/5G_APIs-overrides/TS26512_CommonData.yaml:32]: Could we tighten the syntax up by adding a `pattern` property with a regular expression?

File: 5gms/5G_APIs-overrides/TS26512_CommonData.yaml [hunk 4]
```diff
@@ -0,0 +1,244 @@
...
+      format: uri-reference
+      description: Uniform Resource Locator, conforming with the "URI-reference" production specified in IETF RFC 3986, section 4.1.
+ 
+    RelativeUrl:
+      type: string
```
Comment (rjb1000) [inline 5gms/5G_APIs-overrides/TS26512_CommonData.yaml:37]: Could we tighten the syntax up by adding a `pattern` property with a regular expression?

File: 5gms/5G_APIs-overrides/TS26512_CommonData.yaml [hunk 5]
```diff
@@ -0,0 +1,244 @@
...
+      format: uri-reference
+      description: Relative Uniform Resource Locator, conforming with the "relative-ref" production specified in IETF RFC 3986, section 4.2.
+
+    AbsoluteUrl:
+      type: string
```
Comment (rjb1000) [inline 5gms/5G_APIs-overrides/TS26512_CommonData.yaml:42]: Could we tighten the syntax up by adding a `pattern` property with a regular expression?

**Pull request merged**